VirtoCommerce.Platform.Web/Model/Diagnostics/PlatformInfo.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Platform.Web.Model.Diagnostics
+{
+    public class PlatformInfo
```

Comment:
I think this class is redundant, you can use simple string for platform version representation